<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainHeader</class>
 <widget class="QWidget" name="MainHeader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>38</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>38</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>42</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="HeaderLeft">
     <property name="minimumSize">
      <size>
       <width>42</width>
       <height>42</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>42</width>
       <height>42</height>
      </size>
     </property>
     <property name="toolTip">
      <string>导航</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
   background-color: rgb(45, 120, 82);
   background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(45, 120, 82, 255), stop:0.5 rgba(65, 190, 82, 255), stop:1 rgba(65, 118, 82, 255));

    /*border: none;
    color: white;*/
    border-style: outset;
    border-width: 1px;
    border-radius: 5px;
    border-color: rgb(76, 148, 31);
    padding: 1px;
}

QPushButton:pressed{
    background-color: rgb(76, 148, 31);
    color: white;
}

QPushButton:disabled{
    background-color: gray;
}

QPushButton#checkable{
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(11, 158, 12, 255), stop:1 rgba(150, 220, 120, 255));
    border: none;
    color: white;
}

QPushButton#checkable:hover{
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(11, 158, 12, 255), stop:0.5 rgba(150, 220, 120, 255), stop:1 rgba(11, 158, 12, 255));
    color: white;
    border-left: 5px solid rgb(255, 127, 39);
}

QPushButton#checkable:checked{
    background-color: rgba(11, 158, 12, 255);
    color: white;
    border-left: 5px solid rgb(255, 127, 39);
}

QPushButton#checkable:pressed{
    background-color: rgba(11, 158, 12, 255);
    color: white;
}

QPushButton#checkable:disabled{
    background-color: gray;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource_image.qrc">
       <normaloff>:/image/resource_image/drawer.png</normaloff>:/image/resource_image/drawer.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Icon</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>电抗服务监控</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="HeaderRight">
     <property name="minimumSize">
      <size>
       <width>42</width>
       <height>38</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>42</width>
       <height>38</height>
      </size>
     </property>
     <property name="toolTip">
      <string>关于</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
   background-color: rgb(45, 120, 82);
   background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(24,32,46, 255), stop:0.5 rgba(24,32,46, 230), stop:1 rgba(24,32,46, 255));

    /*border: none;
    color: white;*/
    border-style: outset;
    border-width: 1px;
    border-radius: 12px;
    border-color: rgb(75,132,146,);
    padding: 1px;
}

QPushButton:pressed{
    background-color: rgb(76,32,32);
    color: white;
}

QPushButton:disabled{
    background-color: gray;
}

QPushButton#checkable{
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20,54,44, 233), stop:1 rgba(120,54,44, 255));
    border: none;
    color: white;
}

QPushButton#checkable:hover{
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(24,32,46, 230), stop:0.5 rgba(24,32,46, 230), stop:1 rgba(24,32,46, 230));
    color: white;
    border-left: 5px solid rgb(24,32,46);
}

QPushButton#checkable:checked{
    background-color: rgba(24,32,46, 230);
    color: white;
    border-left: 5px solid rgba(24,32,46, 230);
}

QPushButton#checkable:pressed{
    background-color: rgba(24,32,46, 230);
    color: white;
}

QPushButton#checkable:disabled{
    background-color: gray;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource_image.qrc">
       <normaloff>:/image/resource_image/menu.png</normaloff>:/image/resource_image/menu.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource_image.qrc"/>
 </resources>
 <connections/>
</ui>
